src/store-api/src/region_request.rs

```diff
@@ -473,12 +484,47 @@ impl TryFrom<alter_request::Kind> for AlterKind {
                 let names = x.drop_columns.into_iter().map(|x| x.name).collect();
                 AlterKind::DropColumns { names }
             }
+            Kind::ChangeTableOptions(change_options) => AlterKind::ChangeRegionOptions {
+                options: change_options
+                    .change_table_options
+                    .iter()
+                    .map(TryFrom::try_from)
+                    .collect::<Result<Vec<_>>>()?,
+            },
         };
 
         Ok(alter_kind)
     }
 }
 
+impl TryFrom<&v1::ChangeTableOption> for ChangeRegionOption {
+    type Error = MetadataError;
+
+    fn try_from(option: &v1::ChangeTableOption) -> std::result::Result<Self, Self::Error> {
+        if option.key == TTL_KEY {
+            let ttl_value = if option.value.is_empty() {
+                None
+            } else {
+                let d = humantime::parse_duration(&option.value).map_err(|_| {
```

Comment:
Should we check whether `d` is zero and return None?

Reply:
We now only use zero duration to indicate unset fixed in https://github.com/GreptimeTeam/greptimedb/pull/4848/commits/56f4be4c66137553ad3261fd1163e44d9939bdf7